depends/builders/darwin.mk

```diff
@@ -6,7 +6,7 @@ build_darwin_STRIP:=$(shell xcrun -f strip)
 build_darwin_OBJDUMP:=$(shell xcrun -f objdump)
 build_darwin_NM:=$(shell xcrun -f nm)
 build_darwin_DSYMUTIL:=$(shell xcrun -f dsymutil)
-build_darwin_SHA256SUM=shasum -a 256
+build_darwin_SHA256SUM=sha256sum
```

Comment:
`sha256sum` has never existed by default on macOS afaik. That's actually why this variable exists in the first place. perl or no, shasum is always there.

It's not present for me on Sonoma.

Reply:
Thanks! This change has been dropped.

Reply:
hmmm, it was working for me on Sequoia - it likely comes from `coreutils`, right?

Reply:
Yes, it's from `coreutils`. But shasum exists out of the box.